4001-5000/LIT4957SebhataF.xml

```diff
@@ -1,11 +1,12 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/SChAth/schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT4957SebhataF" xml:lang="en" type="work">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
                 <title xml:lang="gez" xml:id="t1">ስብሐተ፡ ፍቁር፡ ዘጊዮርጊስ፡</title>
                 <title xml:lang="gez" corresp="#t1">Sǝbḥāta fǝqur za-Giyorgis</title>
+                <title xml:lang="gez" corresp="#t1">"Sǝbḥāta fǝqur" of George</title>
```

Comment:
xml:lang="en"
In the Səbḥata fəqur to Gabra Manfas Qəddus, you suggested to have in the translation "Praise of the Beloved of..."

Reply:
Yes. Sorry, now I am a less certain. If "Sebhata fequr" is a kind of hymn, ist is the same case as "Malke'... "? We do not translate Malke'. We can discuss it.